distributed/tensor_parallelism/example.py

```diff
@@ -1,19 +1,22 @@
 import argparse
 import os
+
 import torch
 import torch.distributed as dist
 import torch.multiprocessing as mp
 import torch.nn as nn
 
 TP_AVAILABLE = False
 try:
```

Comment:
I think we should remove those try catch as it's already available?

Reply:
We added version control in requirements. So I think we can remove this.